core/src/trezor/ui/layouts/tt/webauthn.py

```diff
@@ -13,17 +13,24 @@ async def confirm_webauthn(
     ctx: wire.GenericContext | None,
     info: ConfirmInfo,
     pageable: Pageable | None = None,
-) -> bool:
+) -> tuple[bool, int]:
     if pageable is not None:
         confirm: ui.Layout = ConfirmPageable(pageable, ConfirmContent(info))
     else:
         confirm = Confirm(ConfirmContent(info))
 
+    # NOTE: to be compatible with UI2's need to sending a tuple of (is_confirmed, page_index),
+    # sending a mock page_index of "-1", so that the caller knows it should not use it.
     if ctx is None:
-        return is_confirmed(await confirm)
+        return is_confirmed(await confirm), -1
```

Comment:
Why not just get it with `pageable.page()`?

Reply:
Good idea, done in [b557d76](https://github.com/trezor/trezor-firmware/pull/2556/commits/b557d76468ea90567cba981b4288e96502a5b73f)